language-plutus-core/bench/Bench.hs

```diff
@@ -3,10 +3,20 @@ module Main (main) where
 import           Codec.Serialise
 import           Control.Monad
 import           Criterion.Main
-import qualified Data.ByteString.Lazy       as BSL
+import qualified Data.ByteString.Lazy                 as BSL
+import qualified Data.Map                             as M
 import           Language.PlutusCore
+import           Language.PlutusCore.Constant
+import           Language.PlutusCore.Constant.Dynamic
 import           Language.PlutusCore.Pretty
 
+traceBuiltins :: QuoteT (Either (Error ())) DynamicBuiltinNameTypes
```

Comment:
Better to take `getStringBuiltinTypes` from `Language.PlutusTx.Plugin` and put it somewhere common in `language-plutus-core`. Note that it does what @effectfully said below and has a fake definition of `trace`. I don't mind whether we use that or the one in terms of `Debug.trace`.